pyomo/core/tests/unit/test_con.py

```diff
@@ -276,21 +324,23 @@ def rule(model):
         self.assertEqual(model.c.upper, 0)
 
     def test_expr_construct_inf_equality(self):
-        model = self.create_model(abstract=True)
-
-        def rule(model):
-            return model.x == float('inf')
-
-        model.c = Constraint(rule=rule)
-        self.assertRaises(ValueError, model.create_instance)
+        model = self.create_model(abstract=True).create_instance()
```

Comment:
Same question about just using `ConcreteModel`.

Reply:
Same as above: we certainly could change these tests, but I didn't to minimize (unnecessary) changes to old code.